SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/OntologyMapper.Mapped.csproj

```diff
@@ -8,8 +8,8 @@
 		<Version>$(VersionPrefix)</Version>
 		<Authors>Microsoft</Authors>
 		<Description>Support for converting from one Mapped Ontology to a different DTDL Based Ontology</Description>
-		<AssemblyVersion>0.10.9</AssemblyVersion>
-		<PackageVersion>0.10.9-preview</PackageVersion>
+		<AssemblyVersion>0.10.10</AssemblyVersion>
```

Comment:
I'm trying to determine a way to improve this process, but 0.10.10 is actually the current version, can you please rev one more to 0.10.11? After that I'll merge.

Reply:
@Tyler-Angell sure thing. Just pushed the update. Thanks!